INFO 2023-02-17 18:50:24,793 autoreload 13212 7348 Watching for file changes with StatReloader
INFO 2023-02-17 18:51:03,239 autoreload 13212 7348 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\accounts\api.py changed, reloading.
INFO 2023-02-17 18:51:04,723 autoreload 12856 16056 Watching for file changes with StatReloader
INFO 2023-02-20 15:48:38,363 autoreload 9044 11652 Watching for file changes with StatReloader
INFO 2023-02-27 16:42:36,723 autoreload 10564 21264 Watching for file changes with StatReloader
INFO 2023-02-27 16:43:36,387 autoreload 15756 21020 Watching for file changes with StatReloader
WARNING 2023-02-27 16:43:40,520 log 15756 13464 Not Found: /
WARNING 2023-02-27 16:43:40,521 basehttp 15756 13464 "GET / HTTP/1.1" 404 3101
WARNING 2023-02-27 16:43:40,660 log 15756 13464 Not Found: /favicon.ico
WARNING 2023-02-27 16:43:40,660 basehttp 15756 13464 "GET /favicon.ico HTTP/1.1" 404 3152
INFO 2023-02-27 16:43:47,286 basehttp 15756 13464 "GET /admin/ HTTP/1.1" 200 8053
INFO 2023-02-27 16:43:47,378 basehttp 15756 13464 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,379 basehttp 15756 12432 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,380 basehttp 15756 7940 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,381 basehttp 15756 7632 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,420 basehttp 15756 12432 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,420 basehttp 15756 13464 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,477 basehttp 15756 12432 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,496 basehttp 15756 12432 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,497 basehttp 15756 13464 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,543 basehttp 15756 7940 "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,646 basehttp 15756 7940 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,647 basehttp 15756 13464 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2023-02-27 16:43:47,651 basehttp 15756 12432 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,493 basehttp 15756 12432 "GET /admin/home/accepted/ HTTP/1.1" 200 12185
INFO 2023-02-27 16:43:49,579 basehttp 15756 12432 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,580 basehttp 15756 7940 "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,582 basehttp 15756 15332 "GET /static/admin/js/core.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,582 basehttp 15756 13464 "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,587 basehttp 15756 12432 "GET /static/admin/js/actions.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,588 basehttp 15756 7940 "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,589 basehttp 15756 13968 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,594 basehttp 15756 13464 "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,594 basehttp 15756 15332 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,771 basehttp 15756 13968 "GET /static/admin/js/filters.js HTTP/1.1" 304 0
INFO 2023-02-27 16:43:49,901 basehttp 15756 7632 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 16:43:49,949 basehttp 15756 7632 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
INFO 2023-02-27 16:43:54,197 basehttp 15756 7632 "GET /admin/ HTTP/1.1" 200 8053
INFO 2023-02-27 16:46:49,779 autoreload 13968 7632 Watching for file changes with StatReloader
WARNING 2023-02-27 16:46:51,153 log 13968 16500 Not Found: /
WARNING 2023-02-27 16:46:51,154 basehttp 13968 16500 "GET / HTTP/1.1" 404 3101
INFO 2023-02-27 16:46:58,210 basehttp 13968 16500 "GET /admin/ HTTP/1.1" 200 8053
INFO 2023-02-27 16:47:00,765 basehttp 13968 16500 "GET /admin/home/accepted/ HTTP/1.1" 200 12185
INFO 2023-02-27 16:47:01,256 basehttp 13968 16500 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 17:02:09,632 autoreload 20120 10856 Watching for file changes with StatReloader
INFO 2023-02-27 17:10:34,647 autoreload 16924 1892 Watching for file changes with StatReloader
WARNING 2023-02-27 17:10:36,864 log 16924 5024 Not Found: /
WARNING 2023-02-27 17:10:36,865 basehttp 16924 5024 "GET / HTTP/1.1" 404 3101
WARNING 2023-02-27 17:10:37,131 log 16924 5024 Not Found: /favicon.ico
WARNING 2023-02-27 17:10:37,131 basehttp 16924 5024 "GET /favicon.ico HTTP/1.1" 404 3152
INFO 2023-02-27 17:10:43,496 basehttp 16924 5024 "GET /admin/ HTTP/1.1" 200 8053
INFO 2023-02-27 17:10:45,402 basehttp 16924 5024 "GET /admin/home/accepted/ HTTP/1.1" 200 12506
INFO 2023-02-27 17:10:45,490 basehttp 16924 13452 "GET /static/admin/img/search.svg HTTP/1.1" 304 0
INFO 2023-02-27 17:10:45,914 basehttp 16924 5024 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 17:10:51,136 basehttp 16924 5024 "GET /admin/home/accepted/?q=rat HTTP/1.1" 200 8891
INFO 2023-02-27 17:10:51,500 basehttp 16924 5024 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 17:10:57,469 basehttp 16924 5024 "GET /admin/home/accepted/?q=greet HTTP/1.1" 200 8909
INFO 2023-02-27 17:10:57,949 basehttp 16924 5024 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 17:11:00,364 basehttp 16924 5024 "GET /admin/home/accepted/?q=rat HTTP/1.1" 200 8891
INFO 2023-02-27 17:11:04,991 basehttp 16924 5024 "GET /admin/home/accepted/ HTTP/1.1" 200 12506
INFO 2023-02-27 17:45:49,321 autoreload 16268 20696 Watching for file changes with StatReloader
INFO 2023-02-27 17:45:52,350 basehttp 16268 17420 "POST /api/add-after-search HTTP/1.1" 200 17
INFO 2023-02-27 17:46:00,247 basehttp 16268 17420 "POST /api/add-after-search HTTP/1.1" 200 15
INFO 2023-02-27 17:46:06,247 basehttp 16268 20212 "GET /admin/home/accepted/ HTTP/1.1" 200 12506
INFO 2023-02-27 17:46:06,691 basehttp 16268 20212 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
WARNING 2023-02-27 17:46:07,020 log 16268 20212 Not Found: /favicon.ico
WARNING 2023-02-27 17:46:07,020 basehttp 16268 20212 "GET /favicon.ico HTTP/1.1" 404 3344
INFO 2023-02-27 17:46:09,710 basehttp 16268 20212 "GET /admin/home/rhymes/ HTTP/1.1" 200 8943
INFO 2023-02-27 17:46:10,196 basehttp 16268 20212 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 17:51:22,140 autoreload 16432 5876 Watching for file changes with StatReloader
WARNING 2023-02-27 17:51:23,389 log 16432 6332 Not Found: /
WARNING 2023-02-27 17:51:23,389 basehttp 16432 6332 "GET / HTTP/1.1" 404 3293
WARNING 2023-02-27 17:51:23,535 log 16432 6332 Not Found: /favicon.ico
WARNING 2023-02-27 17:51:23,536 basehttp 16432 6332 "GET /favicon.ico HTTP/1.1" 404 3344
WARNING 2023-02-27 17:51:34,440 log 16432 6332 Method Not Allowed: /api/add-after-search
WARNING 2023-02-27 17:51:34,441 basehttp 16432 6332 "GET /api/add-after-search HTTP/1.1" 405 5668
INFO 2023-02-27 17:51:34,648 basehttp 16432 12268 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2023-02-27 17:51:34,654 basehttp 16432 7556 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2023-02-27 17:51:34,656 basehttp 16432 12268 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
INFO 2023-02-27 17:51:34,669 basehttp 16432 12268 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2023-02-27 17:51:34,670 basehttp 16432 7556 "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 200 89476
INFO 2023-02-27 17:51:34,670 basehttp 16432 6332 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
INFO 2023-02-27 17:51:34,692 basehttp 16432 6332 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1719
INFO 2023-02-27 17:51:34,707 basehttp 16432 12268 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
INFO 2023-02-27 17:51:34,725 basehttp 16432 6332 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2023-02-27 17:51:34,726 basehttp 16432 7556 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
INFO 2023-02-27 17:51:34,741 basehttp 16432 7556 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2023-02-27 17:57:25,000 basehttp 16432 15524 "GET /admin/home/rhymes/ HTTP/1.1" 200 8943
INFO 2023-02-27 17:57:25,453 basehttp 16432 15524 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
WARNING 2023-02-27 17:57:25,539 log 16432 15524 Not Found: /favicon.ico
WARNING 2023-02-27 17:57:25,540 basehttp 16432 15524 "GET /favicon.ico HTTP/1.1" 404 3344
INFO 2023-02-27 17:57:27,444 basehttp 16432 15524 "GET /admin/home/rhymes/ HTTP/1.1" 200 8943
INFO 2023-02-27 17:57:27,941 basehttp 16432 15524 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 18:27:59,477 basehttp 16432 416 "POST /api/add-after-search HTTP/1.1" 200 5618
INFO 2023-02-27 18:28:31,392 basehttp 16432 416 "GET /static/rest_framework/css/bootstrap.min.css.map HTTP/1.1" 200 540434
INFO 2023-02-27 20:09:03,619 basehttp 16432 13612 "GET /admin/home/rhymes/ HTTP/1.1" 200 9256
INFO 2023-02-27 20:09:04,106 basehttp 16432 13612 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-02-27 20:32:24,292 basehttp 16432 15048 "GET /admin/home/rhymes/ HTTP/1.1" 200 9256
INFO 2023-02-27 20:32:24,773 basehttp 16432 15048 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-03-02 17:23:04,903 autoreload 6424 20296 Watching for file changes with StatReloader
WARNING 2023-03-02 17:23:51,120 log 6424 7308 Not Found: /api/search-rhyming-words
WARNING 2023-03-02 17:23:51,121 basehttp 6424 7308 "GET /api/search-rhyming-words HTTP/1.1" 404 3959
INFO 2023-03-02 17:24:07,869 autoreload 6424 20296 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:24:09,488 autoreload 16100 9392 Watching for file changes with StatReloader
INFO 2023-03-02 17:24:18,295 autoreload 16100 9392 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\urls.py changed, reloading.
INFO 2023-03-02 17:24:19,776 autoreload 384 17928 Watching for file changes with StatReloader
ERROR 2023-03-02 17:24:21,167 log 384 20980 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print(list(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:24:21,169 basehttp 384 20980 "GET /api/search-rhyming-words HTTP/1.1" 500 94988
ERROR 2023-03-02 17:24:23,220 log 384 20980 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print(list(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:24:23,221 basehttp 384 20980 "GET /api/search-rhyming-words HTTP/1.1" 500 94988
INFO 2023-03-02 17:24:50,970 autoreload 384 17928 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:24:52,509 autoreload 11064 16576 Watching for file changes with StatReloader
ERROR 2023-03-02 17:25:03,065 log 11064 19932 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print(list(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:25:03,067 basehttp 11064 19932 "GET /api/search-rhyming-words HTTP/1.1" 500 94992
INFO 2023-03-02 17:25:12,347 autoreload 11064 16576 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:25:13,497 autoreload 21444 14276 Watching for file changes with StatReloader
INFO 2023-03-02 17:26:20,418 autoreload 21444 14276 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:26:21,876 autoreload 12592 9876 Watching for file changes with StatReloader
INFO 2023-03-02 17:26:24,496 basehttp 12592 20192 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:26:38,352 autoreload 12592 9876 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:26:39,602 autoreload 11568 9320 Watching for file changes with StatReloader
INFO 2023-03-02 17:26:47,403 basehttp 11568 20424 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:26:48,580 basehttp 11568 20424 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:27:48,103 autoreload 11568 9320 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:27:49,506 autoreload 14800 12724 Watching for file changes with StatReloader
INFO 2023-03-02 17:28:55,807 autoreload 14800 12724 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:28:57,343 autoreload 624 7660 Watching for file changes with StatReloader
ERROR 2023-03-02 17:29:00,199 log 624 2444 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching",list(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:29:00,201 basehttp 624 2444 "GET /api/search-rhyming-words HTTP/1.1" 500 95046
INFO 2023-03-02 17:29:16,427 autoreload 624 7660 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:29:18,077 autoreload 9036 18224 Watching for file changes with StatReloader
ERROR 2023-03-02 17:29:21,449 log 9036 17212 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching",dict(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:29:21,450 basehttp 9036 17212 "GET /api/search-rhyming-words HTTP/1.1" 500 95046
INFO 2023-03-02 17:32:45,392 autoreload 9036 18224 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:32:47,298 autoreload 3724 7356 Watching for file changes with StatReloader
ERROR 2023-03-02 17:32:49,891 log 3724 16860 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching",i.values)
AttributeError: 'Accepted' object has no attribute 'values'
ERROR 2023-03-02 17:32:49,893 basehttp 3724 16860 "GET /api/search-rhyming-words HTTP/1.1" 500 95208
INFO 2023-03-02 17:33:02,352 autoreload 3724 7356 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:33:03,865 autoreload 6832 12272 Watching for file changes with StatReloader
INFO 2023-03-02 17:33:10,500 basehttp 6832 15568 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:33:28,191 autoreload 6832 12272 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:33:29,462 autoreload 16964 6956 Watching for file changes with StatReloader
INFO 2023-03-02 17:33:31,792 basehttp 16964 11040 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:33:43,455 autoreload 16964 6956 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:33:44,934 autoreload 2368 6772 Watching for file changes with StatReloader
ERROR 2023-03-02 17:33:45,941 log 2368 17060 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching",i.data)
AttributeError: 'Accepted' object has no attribute 'data'
ERROR 2023-03-02 17:33:45,943 basehttp 2368 17060 "GET /api/search-rhyming-words HTTP/1.1" 500 95192
INFO 2023-03-02 17:33:53,919 autoreload 2368 6772 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:33:55,412 autoreload 17036 1320 Watching for file changes with StatReloader
INFO 2023-03-02 17:35:21,306 autoreload 17036 1320 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:35:22,816 autoreload 19460 20556 Watching for file changes with StatReloader
ERROR 2023-03-02 17:35:24,100 log 19460 17100 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching",i.values())
AttributeError: 'Accepted' object has no attribute 'values'
ERROR 2023-03-02 17:35:24,103 basehttp 19460 17100 "GET /api/search-rhyming-words HTTP/1.1" 500 95212
INFO 2023-03-02 17:35:39,902 autoreload 19460 20556 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:35:41,289 autoreload 18512 20620 Watching for file changes with StatReloader
ERROR 2023-03-02 17:35:43,117 log 18512 21168 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching",list(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:35:43,119 basehttp 18512 21168 "GET /api/search-rhyming-words HTTP/1.1" 500 95046
INFO 2023-03-02 17:37:07,881 autoreload 18512 20620 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:37:08,958 autoreload 18204 11604 Watching for file changes with StatReloader
ERROR 2023-03-02 17:37:11,403 log 18204 13892 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 172, in get
    for i in range(user_info):
TypeError: 'QuerySet' object cannot be interpreted as an integer
ERROR 2023-03-02 17:37:11,406 basehttp 18204 13892 "GET /api/search-rhyming-words HTTP/1.1" 500 94304
ERROR 2023-03-02 17:37:12,509 log 18204 13892 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 172, in get
    for i in range(user_info):
TypeError: 'QuerySet' object cannot be interpreted as an integer
ERROR 2023-03-02 17:37:12,511 basehttp 18204 13892 "GET /api/search-rhyming-words HTTP/1.1" 500 94304
INFO 2023-03-02 17:37:20,950 autoreload 18204 11604 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:37:22,790 autoreload 1468 11828 Watching for file changes with StatReloader
ERROR 2023-03-02 17:37:23,153 log 1468 16400 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 172, in get
    for i in range(user_info):
TypeError: 'QuerySet' object cannot be interpreted as an integer
ERROR 2023-03-02 17:37:23,155 basehttp 1468 16400 "GET /api/search-rhyming-words HTTP/1.1" 500 94300
ERROR 2023-03-02 17:37:24,318 log 1468 16400 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 172, in get
    for i in range(user_info):
TypeError: 'QuerySet' object cannot be interpreted as an integer
ERROR 2023-03-02 17:37:24,319 basehttp 1468 16400 "GET /api/search-rhyming-words HTTP/1.1" 500 94300
ERROR 2023-03-02 17:37:24,864 log 1468 16400 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 172, in get
    for i in range(user_info):
TypeError: 'QuerySet' object cannot be interpreted as an integer
ERROR 2023-03-02 17:37:24,866 basehttp 1468 16400 "GET /api/search-rhyming-words HTTP/1.1" 500 94300
INFO 2023-03-02 17:38:59,031 autoreload 1468 11828 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:39:00,476 autoreload 18264 13572 Watching for file changes with StatReloader
INFO 2023-03-02 17:39:09,062 autoreload 18264 13572 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:39:10,234 autoreload 15876 15808 Watching for file changes with StatReloader
INFO 2023-03-02 17:39:38,718 autoreload 15876 15808 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:39:40,123 autoreload 17160 7412 Watching for file changes with StatReloader
ERROR 2023-03-02 17:39:44,239 log 17160 9336 Internal Server Error: /api/search-rhyming-words
Traceback (most recent call last):
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\django\views\generic\base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\venv\lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py", line 174, in get
    print("Inside searching", dict(i))
TypeError: 'Accepted' object is not iterable
ERROR 2023-03-02 17:39:44,241 basehttp 17160 9336 "GET /api/search-rhyming-words HTTP/1.1" 500 95048
INFO 2023-03-02 17:39:52,063 autoreload 17160 7412 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:39:53,504 autoreload 15764 1900 Watching for file changes with StatReloader
INFO 2023-03-02 17:39:54,864 basehttp 15764 16424 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:42:53,776 autoreload 15764 1900 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:42:55,192 autoreload 9060 19532 Watching for file changes with StatReloader
INFO 2023-03-02 17:43:01,060 basehttp 9060 8740 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 17:45:06,356 autoreload 9060 19532 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 17:45:08,085 autoreload 12048 14412 Watching for file changes with StatReloader
INFO 2023-03-02 17:45:13,132 basehttp 12048 8448 "GET /api/search-rhyming-words HTTP/1.1" 200 1320
INFO 2023-03-02 18:25:15,734 autoreload 12048 14412 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 18:25:17,482 autoreload 12032 1508 Watching for file changes with StatReloader
INFO 2023-03-02 20:00:31,121 autoreload 12032 1508 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\models.py changed, reloading.
INFO 2023-03-02 20:00:33,452 autoreload 6072 15764 Watching for file changes with StatReloader
INFO 2023-03-02 20:22:34,737 autoreload 6072 15764 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\models.py changed, reloading.
INFO 2023-03-02 20:22:36,057 autoreload 16308 11180 Watching for file changes with StatReloader
INFO 2023-03-02 20:22:54,660 autoreload 16308 11180 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\models.py changed, reloading.
INFO 2023-03-02 20:22:56,179 autoreload 19336 15228 Watching for file changes with StatReloader
INFO 2023-03-02 20:24:30,516 autoreload 19336 15228 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\admin.py changed, reloading.
INFO 2023-03-02 20:24:32,055 autoreload 7512 14352 Watching for file changes with StatReloader
INFO 2023-03-02 20:24:53,440 autoreload 7512 14352 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\admin.py changed, reloading.
INFO 2023-03-02 20:24:54,664 autoreload 9868 15520 Watching for file changes with StatReloader
INFO 2023-03-02 20:25:19,211 autoreload 9868 15520 C:\VE\000_Client\Blake_Dictionary_Working - Copy\backend\home\views.py changed, reloading.
INFO 2023-03-02 20:25:20,393 autoreload 3612 20300 Watching for file changes with StatReloader
INFO 2023-03-02 20:31:47,079 autoreload 17632 2668 Watching for file changes with StatReloader
INFO 2023-03-02 20:32:29,638 autoreload 1348 19572 Watching for file changes with StatReloader
WARNING 2023-03-02 20:32:32,296 log 1348 10752 Not Found: /
WARNING 2023-03-02 20:32:32,297 basehttp 1348 10752 "GET / HTTP/1.1" 404 3293
WARNING 2023-03-02 20:32:32,464 log 1348 10752 Not Found: /favicon.ico
WARNING 2023-03-02 20:32:32,465 basehttp 1348 10752 "GET /favicon.ico HTTP/1.1" 404 3344
INFO 2023-03-02 20:32:37,923 basehttp 1348 10752 "GET /admin/ HTTP/1.1" 200 8493
INFO 2023-03-02 20:32:37,971 basehttp 1348 10752 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2023-03-02 20:32:37,971 basehttp 1348 8644 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2023-03-02 20:32:37,978 basehttp 1348 8644 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2023-03-02 20:32:37,978 basehttp 1348 10752 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2023-03-02 20:32:37,985 basehttp 1348 21304 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,085 basehttp 1348 21304 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,085 basehttp 1348 8644 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,107 basehttp 1348 21304 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,107 basehttp 1348 8644 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,113 basehttp 1348 21304 "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,291 basehttp 1348 8644 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,291 basehttp 1348 21304 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2023-03-02 20:32:38,292 basehttp 1348 10752 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
